src/assets/logos/sponsors/Redfin.svg

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
don't have to remove the actual image.... you can just remove them  from the jason file

Reply:
I wanted to make sure that none of the images are accessible from a user using inspect element since the sponsors are a surprise